Example/Backpack/UIKit/Components/Rating/RatingsViewController.swift

```diff
@@ -15,6 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import Backpack_Common
```

Comment:
@frugoman we need this import here because the clamping logic has moved 'up' into common. Do you feel this is a good trade-off? It's nice we have centralized logic, but maybe we want to make sure consumers don't have to import this if they are using the BPKRatingScale.

@dennis-li-skyscanner we could consider making a 'mapping' inside Backpack/Backpack-SwiftUI so that consumers don't need to import the common package. 

Then we can map Backpack.RatingScale -> Common.RatingScale. 

Let me know what you think

Reply:
Two mapping enums are created. And the implementation is located in BPKCommonRatingScale.